game.py

```diff
@@ -0,0 +1,150 @@
+import random
+import string
+
+
+class Game:
+    def __init__(self, size=3, target=100, level='medium'):
+        self.size = size
+        self.target = target
+        self.level = level
+        self.characters = string.ascii_letters
+        self.board = [[None] * self.size for _ in range(size)]
+
+    def set_up_game(self):
+        self.set_mode()
+        self.set_values()
+        self.score = 0
+        self.winner = False
+        self.set_up_board()
+
+    def set_mode(self):
+        if self.level == 'easy':
+            self.characters = self.characters[:self.size - self.size // 2]
+        elif self.level == 'medium':
+            self.characters = self.characters[:self.size - self.size // 3]
+        elif self.level == 'hard':
+            self.characters = self.characters[:self.size]
+
+    def set_values(self):
+        self.delete = set()
```

Comment:
Shouldnt instance variables be declared in teh `__init__`.
It's also weird that `delete` `create` are instance variables, thsi maybe requires an offline discussion...

Reply:
I replaced self.delete and self.create by local variables. Thanks for the offline discussion :)